src/composables/useAreaOfInterest.ts

```diff
@@ -21,8 +21,10 @@ import { showWarning } from '../functions/snackbar'
 import { booleanWithin as turfBooleanWithin } from '@turf/boolean-within'
 import { useSearch, type SearchResults } from './useSearch'
 import { Feature as GeoJSONFeature, Polygon as GeoJSONPolygon } from 'geojson'
+import { usePermalink } from './usePermalink'
 
 const { clearSearchResults } = useSearch()
+const { updateTileSelection } = usePermalink()
```

Comment:
It is not good practice to call `use*()` functions of a composable on the module level of another composable, because `use*()` functions often require a `<script setup>` context, which is not available on the module level of another composable.

As a quick fix, `export` the `updateTileSelection()` function from `usePermalink.ts` to indicate it does not require q `<script setup>` context, and import it here directly. Same with `clearSearchResults()` from `useSearch.ts` - I had missed that in the review of the pull request where that was introduced, sorry for that.